### ISSUE TITLE
Uncaught TypeError: other.getPath is not a function

### ISSUE THREAD

**Issue by aurovrata** [gwomacks/php-debug#251]
I keep getting this error if I try to setup more than one breakpoint.

The first bp is registered and xdbug works fine.  If I try to add another bp I get this error,
`
Uncaught TypeError: other.getPath is not a function`

**Comment by aurovrata** [comment]
in fact, I can remove the working bp and then add one elsewhere and it works again, but as soon as I try to add another bp, the error pops up again

**Comment by cchamplin** [comment]
Can you check if this is resolved in 0.2.6?

**Comment by aurovrata** [comment]
no, still an issue.  I am on a linux machine,
```
php v7.0
$ atom --version
Atom    : 1.22.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
php-debug v0.2.6
stack trace:
~/.atom/packages/php-debug/lib/models/breakpoint.coffee:95
Hide Stack Trace
TypeError: other.getPath is not a function
    at Breakpoint.module.exports.Breakpoint.isEqual (~/.atom/packages/php-debug/lib/models/breakpoint.coffee:95:27)
    at Object.exports.arraySearch (~/.atom/packages/php-debug/lib/helpers.coffee:32:15)
    at Object.exports.arrayRemove (~/.atom/packages/php-debug/lib/helpers.coffee:37:19)
    at GlobalContext.module.exports.GlobalContext.removeBreakpoint (~/.atom/packages/php-debug/lib/models/global-context.coffee:44:23)
    at Object.toggleBreakpoint (~/.atom/packages/php-debug/lib/php-debug.coffee:492:30)
    at HTMLDivElement.<anonymous> (~/.atom/packages/php-debug/lib/php-debug.coffee:408:8)
    at HTMLDivElement.dispatch (~/.atom/packages/php-debug/node_modules/jquery/dist/jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (~/.atom/packages/php-debug/node_modules/jquery/dist/jquery.js:4121:28)
```

**Comment by cchamplin** [comment]
This is fairly odd, what you're describing shouldn't really be possible (there are specific code checks for it). I'm wondering if it's possibly some type of data corruption in the local storage for the plugin. We might have to try a few things.

Can you clear all breakpoints via Packages->PHP Debug->Clear All Breakpoints, then restart Atom. Let me know if that fixes it.

**Comment by AyeAyePhyuKyaw** [comment]
If you install wordpress-suite 1.2.1 package, you face "TypeError: other.getPath is not a function" error and cannot toggle more than one breakpoint.
I also faced same error. I solved it by disable wordpress-site 1.2.1.

I hope you will be ok.

**Comment by aurovrata** [comment]
@AyeAyePhyuKyaw bingo, had to remove the package altogether to get the bp to work.

**Comment by cchamplin** [comment]
Should be fixed in v0.3.0
